apps/ledger-live-desktop/src/renderer/modals/UpdateFirmwareModal/steps/02-step-flash-mcu.jsx

```diff
@@ -60,9 +67,49 @@ const StepFlashMcu = ({ firmware, deviceModelId, setError, transitionTo }: Props
   const [progress, setProgress] = useState(0);
   const withFinal = useMemo(() => hasFinalFirmware(firmware?.final), [firmware]);
 
+  const [isMcuUpdateFinished, setIsMcuUpdateFinished] = useState<boolean>(false);
   const deviceLocalizationFeatureFlag = useFeature("deviceLocalization");
 
-  // didMount
+  // Gets the updated device info from the command firmwareUpdating
+  // after a successful MCU update
+  useEffect(() => {
+    let sub;
+
+    if (isMcuUpdateFinished) {
+      sub = (getEnv("MOCK") ? mockedEventEmitter() : command("firmwareUpdating")({ deviceId: "" }))
```

Comment:
i would just suggest to rename the whole command from `firmwareUpdating` maybe to `waitForDeviceInfo` as we start to need this for more than the "firmwareUpdating case.

Reply:
Agreed -> done, but **i did not change** the names of the step `02-step-updating.tsx (d47cc84)` and its associated [renderFirmwareUpdating](https://github.com/LedgerHQ/ledger-live/blob/380086b2fef163a1fc7d19fe072783410a72cb47/apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx#L619) and illustrations